packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -15,7 +15,6 @@
   margin-block-end: $spacing-xs-em;
   margin-inline: 0;
   color: $text-heading;
```

Comment:
Can this be removed? [Box provides color](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/src/components/Box/Box.tsx#L303) although nothing specific to heading. We should log an issue for this and prioritize it as part of the design token effort. Thoughts?

Reply:
The issue with deferring this responsibility to `Box` is that it removes the ability to do individual component level theming